src/Symfony/Bundle/WebProfilerBundle/Controller/RouterController.php

```diff
@@ -99,4 +103,26 @@ private function getTraces(RequestDataCollector $request, $method)
 
         return $matcher->getTracesForRequest($traceRequest);
     }
+
+    /**
+     * Replaces scalars wrapped as Data instances by their actual values and removes other values.
```

Comment:
Because keeping the `Data` instances (there can only be such things here really) can only trigger bad behaviors, so removing them looks more safe. We just need to resolve the values to allow the url matcher do to its work. Even if it could match on weird attributes or on array cookies, let's not care about that to keep the code effective.